.chloggen/awsproxy_lifecycle.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: awsproxyextension
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Make awsproxy extension support shutdown
```

Comment:
Out of ignorance, I don't understand how this adds support for shutdown. Shutdown itself already existed, what's the user-facing change that's worth documenting here? 

I assume the lack of a `nil` check before is what blocked the generated component test from being enabled?

Reply:
It would fail at creation time. The message is not great, sorry.

Reply:
```suggestion
note: Make awsproxy extension more resilient by initiating the AWS client during Start.
```